joint_state_controller:
  type: "joint_state_controller/JointStateController"
  publish_rate: 50

wheel_1_joint_controller:
  type: "velocity_controllers/JointVelocityController"
  joint: "l1_joint"
  # pid: {p: 50.0, i: 0.1, d: 1.0}

wheel_2_joint_controller:
  type: "velocity_controllers/JointVelocityController"
  joint: "l2_joint"
  # pid: {p: 50.0, i: 0.1, d: 1.0}

wheel_3_joint_controller:
  type: "velocity_controllers/JointVelocityController"
  joint: "r1_joint"
  # pid: {p: 50.0, i: 0.1, d: 1.0}

wheel_4_joint_controller:
  type: "velocity_controllers/JointVelocityController"
  joint: "r2_joint"
  # pid: {p: 50.0, i: 0.1, d: 1.0}

arm_1_joint_controller:
  type: "position_controllers/JointPositionController"
  joint: "arm_joint_1"
  # pid: {p: 50.0, i: 0.01, d: 0.5}

arm_2_joint_controller:
  type: "position_controllers/JointPositionController"
  joint: "arm_joint_2"
  # pid: {p: 50.0, i: 0.01, d: 0.5}
